targets/arm.ld

```diff
@@ -31,6 +31,17 @@ SECTIONS
         _stack_top = .;
     } >RAM
 
+    /* Persistent RAM that survives reset. */
+    .persist (NOLOAD) :
+    {
+        . = ALIGN(4);
+        _spersist = .;
+        *(.persist)
+        *(.persist.*)
```

Comment:
I've never understood the convention with these includes, are they both necessary? Wouldn't `*(.persist*)`, or, in this case, simply `*(.persist)` be sufficient?

Reply:
It's simply a glob. So `.persist` matches exactly the string `.persist` while `.persist.*` matches anything that starts with `.persist.`. This emulates somewhat the similar conventions around `-ffunction-sections` and `-fdata-sections`.